ranger/container/tags.py

```diff
@@ -103,7 +106,7 @@ def _parse(self, fobj):
         result = dict()
         for line in fobj:
             line = line.rstrip('\n')
-            if len(line) > 2 and line[1] == ':':
+            if len(line) > 2 and line[1] == ':' and line[0] != self.default_tag:
```

Comment:
What's this about?

Reply:
Was testing stuff, forgot to take it out.